src/system/sys_profiler.hh

```diff
@@ -376,8 +451,50 @@ protected:
      * Total number of bytes deallocated directly in this frame (not include subframes).
      */
     size_t total_deallocated_;
-
-
+    /**
+     * Maximum number of bytes allocated at one time in this frame (not include subframes).
+     */
+    size_t max_allocated_;
+    /**
+     * Current number of bytes allocated in this frame (not include subframes).
+     */
+    size_t current_allocated_;
```

Comment:
Do you mean "Total number of bytes allocated"?, i.e. opposite of total_deallocated_. Rename, modify comment.